internal/netpoll/queue/queue.go

```diff
@@ -20,12 +20,30 @@
 
 package queue
 
-// Task is a asynchronous function.
-type Task func() error
+import "sync"
+
+type TaskFunc func([]byte) error
+
+// Task is a wrapper that contains function and its argument.
+type Task struct {
+	Run TaskFunc
```

Comment:
`Run` -> `Func`?

Reply:
I think `Run` is a better match for `Task` than `Func`.